site/docs/v1/tech/apis/scim/_scim-group-response-body.adoc

```diff
@@ -0,0 +1,53 @@
+Because SCIM schemas are completely customizable, The SCIM response returned is generated from the result of SCIM Group object populated by your outgoing response lambda.
+
```

Comment:
Same comment about the mapping and the field definitions.

Reply:
done